backend/README.md

```diff
@@ -30,14 +32,16 @@ Follow the instructions below to run the backend locally. Change directory to `/
 
 1. Set-up a virtual environment
 
-> It's recommended (though not technically required) to create a virtual environment for the project by running `python -m venv .venv` to create it and `.venv/Scripts/activate` to activate it in your active terminal.
+> It's recommended (though not technically required) to create a virtual environment for the project by running `python -m venv .venv` to create it and `. .venv/Scripts/activate` to activate it in your active terminal.
```

Comment:
```suggestion
> It's recommended (though not technically required) to create a virtual environment for the project by running `python -m venv venv` to create it and `source venv/Scripts/activate` to activate it in your active terminal.
```

I would recommend fixing the venv creation command to call the folder `venv` instead of updating the command to run the venv. When the folder isn't hidden I find that you get nicer auto-completion in your CLI when running the venv.

Also suggest this doc recommends `source`, does the same thing but reads nicer